tools/autograd/deprecated.yaml

```diff
@@ -1,92 +1,134 @@
 # Deprecated function signatures. These are exposed in Python, but not included
 # in the error message suggestions.
 
-- name: add(Tensor self, Scalar alpha, Tensor other)
+- name: add(Tensor self, Scalar alpha, Tensor other) -> Tensor
   aten: add(self, other, alpha)
 
-- name: add(Tensor self, Scalar alpha, Tensor other, *, Tensor out)
+- name: add_(Tensor(a!) self, Scalar alpha, Tensor other) -> Tensor(a!)
```

Comment:
The old `load_deprecated_signatures` treated functional an inplace schemas as equivalent, but the matching is much more strict now so these have to be written out explicitly.